dev/preview/infrastructure/gce/cloudinit.yaml

```diff
@@ -0,0 +1,31 @@
+#cloud-config
+users:
+- name: ubuntu
+  sudo: "ALL=(ALL) NOPASSWD: ALL"
+  ssh_authorized_keys:
+    - ${ssh_authorized_keys}
+chpasswd:
+  list: |
+    ubuntu:ubuntu
+  expire: False
+write_files:
+  - path: /usr/local/bin/bootstrap.sh
+    permissions: '0744'
+    owner: root
+    content: |
+      #!/bin/bash
+
+      set -eo pipefail
+
+      sudo systemctl disable load-workspace-full.service &
```

Comment:
Nit: Would love a comment here similar to what you have in the PR description.

As I mention in a comment below, it could be that some users of preview environments will be negatively impacted by the images not being pulled before the first workspace starts, but as GCE-based preview environments are opt-in I think it's fine to keep this as it